content/en/docs/setup/install/operator/test.sh

```diff
@@ -39,6 +39,6 @@ _verify_contains snip_kubectl_get_svc "egressgateway"
 
 # @cleanup
 istioctl operator remove
-istioctl x uninstall --purge -y
+istioctl uninstall -y --purge
 kubectl delete ns istio-system --grace-period=0 --force
```

Comment:
command `x uninstall` does not require confirmation, but `uninstall` does. if we remove -y, the code wont pass the test on ci/cd.

Reply:
I didn't say to remove the -y here, I said to do it in istioctl/index.md, which has no ci/cd test. Even if it did, you could still remove the -y in the doc and make the test work by piping echo y into the snip, the way lots of other tests do it.

Reply:
@frankbu  Using pipe is fine but why not using the command own option instead of having more words to do exactly the same thing? I did notice the pipe used in other places. Using its own option seems to me a better choice regardless it has ci/cd or not, it will work, don't you think that is better?

Reply:
Use -y in the doc (index.md) is used when the command that the doc is shoing doesn't want the user to worry about the prompt. In other docs, it is better to let the user answer the prompt, so in shouldn't include -y. For the second case, if we also have an automated test, then we would use pipe so that the tests will work without user involvement. This does not apply in the istioctl/index.md example where I suggested you remove the -y because there is no automated test for that doc anyway.